File: noteGenerator/src/main/java/com/kjipo/font/PathInterfaceImpl.java
package com.kjipo.font;

import java.util.List;

public class PathInterfaceImpl implements PathInterface {
    private final List<PathElement> pathElements;
    private final int strokeWidth;


    public PathInterfaceImpl(List<PathElement> pathElements, int strokeWidth) {
        this.pathElements = pathElements;
        this.strokeWidth = strokeWidth;
    }


    @Override
    public List<PathElement> getPathElements() {
        return pathElements;
    }

    @Override
    public int getStrokeWidth() {
        return strokeWidth;
    }
}

File: score-common/src/main/kotlin/com/kjipo/svg/PathInterfaceImpl.kt
package com.kjipo.svg

import com.kjipo.score.FILL_COLOUR
import kotlinx.serialization.Serializable

@Serializable
data class PathInterfaceImpl(val pathElements: List<PathElement>,
                        val strokeWidth: Int,
                             val fill:String = FILL_COLOUR)
